Sources/Fullscreen/Stories/StoriesView.swift

```diff
@@ -73,9 +88,16 @@ public class StoriesView: UIView {
 
     // MARK: - Public methods
 
-    public func configure(with stories: [StoryViewModel]) {
+    public func configure(
+        with stories: [StoryViewModel],
+        startIndex: Int,
+        feedbackViewModel: StoryFeedbackViewModel? = nil
+    ) {
         self.stories = stories
+        self.startIndex = startIndex
+        self.feedbackViewModel = feedbackViewModel
         collectionView.reloadData()
+        collectionView.setNeedsLayout()
```

Comment:
I guess you want `self` to have the `layoutSubviews` call? 🙂 
```suggestion
        setNeedsLayout()
```

Reply:
yup, you're right! ✔️